kolibri/core/assets/src/views/ExamReport/PageStatus.vue

```diff
@@ -107,6 +141,42 @@
           return this.$tr('notStartedLabel');
         }
       },
+      retakeEnabled() {
+        // The Retake button is enabled only for leaners,
+        // the threshold should be kept in a proper conf and not here (TODO)
+        var isScoreBelowThreshold =
```

Comment:
looks like this is the only one but make sure you are watching the var vs. let vs. const assignments.